user/udiskie/files/50-udiskie.rules

```diff
@@ -0,0 +1,24 @@
+/**
+ * Polkit permissions for udiskie.
+ * Source: Udiskie README (see https://pypi.python.org/pypi/udiskie)
```

Comment:
don't see it in the README, seems to be here instead now: https://github.com/coldfix/udiskie/wiki/Permissions

also, i don't really know anything about polkit but this file is missing `org.freedesktop.udisks2.encrypted-unlock-system`  and `org.freedesktop.udisks2.filesystem-unmount-others`, which are in the link above, you might want to have these in as well